### ISSUE TITLE
RPC endpoint should be retrieved from Bee

### ISSUE THREAD

**Issue by AuHau** [ethersphere/bee-dashboard#535]
With storing RPC endpoint to a local store (and maybe some other configuration? not sure) and Bee config, there can be a discrepancy between these two when different instance of Swarm Desktop is used. I would suggest then to retrieve it from Bee config instead.

**Comment by mfw78** [comment]
This should be _optional_ with the ability to override the RPC. When compiling `bee-dashboard` separately for use outside of `swarm-desktop`, there may very well be the event where the `bee` node is running using an RPC that is only accessible from the `bee` node's context (such as operating within a docker network, where the user's browser is therefore unable to access the nominated RPC if they use one that is returned from the bee node).

**Comment by AuHau** [comment]
Yeah agree and thanks for reminding it. This issue is scoped to Desktop 😉
